pom.xml

```diff
@@ -45,6 +44,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Je hebt de dependency voor validatie in een DTO toegevoegd, maar niet gebruikt in je DTO. Stond overigens ook niet in de opdracht dat je dit verder moest toepassen.